components/camel-jms/src/test/java/org/apache/camel/component/jms/JmsTopicDurableSharedTest.java

```diff
@@ -42,6 +43,7 @@ void testDurableSharedTopic() throws Exception {
         mock2.expectedMessageCount(1);
         mock2.expectedBodiesReceived("Hello World");
 
+        artemisConsumersWereCreated.await(1000, TimeUnit.MILLISECONDS);
```

Comment:
This latch is noop, as it is just timing out and it's not an adequate use of a latch.

Reply:
Yeah its the same as Thread.sleep if we dont poll artemis for the state of the consumers. Ok, I'll revert